test/files/run/t9915/Test_2.scala

```diff
@@ -0,0 +1,14 @@
+
+object Test extends App {
+  val c = new C_1
+  assert(c.nulled == "X\u0000ABC")
+//assert(c.nulled == "X\000ABC")
```

Comment:
Right, the octal escape is deprecated, but I left it as a comment, since part of the charm of the test was that the two strings were textually the same length.